modules/uhttp/http.go

```diff
@@ -78,7 +78,7 @@ type Config struct {
 	modules.ModuleConfig
 	Port    int           `yaml:"port"`
 	Timeout time.Duration `yaml:"timeout"`
-	Debug   *bool         `yaml:"debug"`
+	Debug   bool          `yaml:"debug"`
```

Comment:
FYI, there are three distinct states for a bool when serializing from yaml: `true`, `false` and `did not specify`, that's why we use `*bool` as opposed to just bool.